ts/Core/Pointer.ts

```diff
@@ -795,8 +795,9 @@ class Pointer {
             [hoverSeries as any] :
             // Filter what series to look in.
             series.filter(function (s): boolean {
-                return eventArgs.filter ? eventArgs.filter(s) : filter(s) &&
-                    s.stickyTracking;
+                return eventArgs.filter ?
+                    eventArgs.filter(s) && s.stickyTracking :
+                    filter(s) && s.stickyTracking;
```

Comment:
Looks like we can trim this expression a bit?
```ts
            series.filter((s): boolean => s.stickyTracking &&
                (eventArgs.filter || filter)(s));
```

Reply:
I think the requested change was applied to the wrong filter function... I requested a change to the filter function on line 798, but it was applied to the one on line 815, probably altering the functionality.